instrumentation/dubbo/src/main/java/brave/dubbo/DubboParser.java

```diff
@@ -19,9 +19,6 @@
 import org.apache.dubbo.rpc.Invoker;
 import org.apache.dubbo.rpc.support.RpcUtils;
 
-import static org.apache.dubbo.rpc.Constants.$INVOKE;
```

Comment:
a separate change would need to make the next version of dubbo not crash tests. this just avoids referencing a constant they later deleted